### ISSUE TITLE
webpack woes (again)

### ISSUE THREAD

**Issue by blacktrash** [flowplayer/flowplayer-hlsjs#43]
- go to live demo at http://demos.flowplayer.org/api/hlsjs.html#hlsjslive
- open console
- start playback
- choose a level
- let run until there's dialogue; may take a while, stream may have to be restarted because it's rather problematic
- console will spew out (ID3 data?) dialogue letter by letter:
<img width="719" alt="screen shot 2016-09-03 at 1 50 19 pm" src="https://cloud.githubusercontent.com/assets/168338/18224947/d3aab8fe-71de-11e6-9dd6-6b072944a562.png">

- does not happen at http://flowplayer.blacktrash.org/fp6/live.html where //cdn.jsdelivr.net/hls.js/0.6.2-5/hls.min.js is loaded separately before flowplayer.hlsjs.js
- does also happen when I back out https://github.com/flowplayer/flowplayer-hlsjs/commit/bfb993d807a08adabefbe92cd7db4c33800a0b1c
Basically when webpack is involved ...

**Comment by blacktrash** [comment]
Also does not happen when I
```sh
wget -O flowplayer.hlsjs.min.js http://cdn.jsdelivr.net/hls.js/0.6.2-5/hls.min.js
uglifyjs -m -c < flowplayer.hlsjs.js >> flowplayer.hlsjs.min.js
```

**Comment by blacktrash** [comment]
Triple checking whether going back to noParse really does not help.

**Comment by blacktrash** [comment]
Indeed noParse also doesn't circumvent this.

**Comment by blacktrash** [comment]
Maybe it is related to (one of) these webpack warnings:
```
Side effects in initialization of unused variable enableLogs [./~/hls.js/lib/utils/logger.js:63,4]
Side effects in initialization of unused variable logger [./~/hls.js/lib/utils/logger.js:81,4]
Side effects in initialization of unused variable ErrorTypes [./~/hls.js/lib/errors.js:6,4]
Side effects in initialization of unused variable ErrorDetails [./~/hls.js/lib/errors.js:15,4]
```
It seems the hls.js error details are spewed out lettter by letter.

**Comment by blacktrash** [comment]
currently testing whether the issue has gone with hls.js v0.6.0.2-6 (https://github.com/dailymotion/hls.js/commit/81681373241bd600e4fd53d1db2271effc96b2ba ???)

**Comment by blacktrash** [comment]
Definitely still happening.
Good time of the day to test http://flowplayer.blacktrash.org/fp6/live.html seems to be 13:00 GMT.

**Comment by nnarhinen** [comment]
This is the most likely candidant: https://github.com/dailymotion/hls.js/blob/d6fda6236b44cf65d0f32deefcd9a3f9897d2a3a/src/utils/cea-608-parser.js#L166-L179

`this.verboseLevel` is read, and `this` might be something different when bundled with Webpack and using CommonJS `require`s than when directly evaluated from within . @mangui can you take a look?

**Comment by blacktrash** [comment]
Indeed webpack brings only the symptoms of this problem to the fore. The same happens when one loads the unminified hls.js release.
webpack deals with the unminified code, and therefore exhibits the problem, which is not apparent when using the minified hls.js distribution.
